onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -1471,7 +1471,15 @@ def _where_input_wrangler(
         dtypes=[torch.float16],
         reason="fixme: ONNX Runtime aborted",
     ),
-    TorchLibOpInfo("nn.functional.linear", nn_ops.aten_linear, trace_only=True),
+    TorchLibOpInfo("nn.functional.linear", nn_ops.aten_linear).skip(
+        # input: input, args: weight, bias; so len(args) == 2 means bias is provided
+        matcher=lambda sample: sample.kwargs.get("bias") is not None or len(sample.args) == 2,
```

Comment:
Do you know why the combination of args/kwargs varies across test samples?

Reply:
So I realized there's only args, no kwargs in this test. Fixed